### DISCUSSION TITLE
Why is my text opacity not changing when disabled?

### DISCUSSION THREAD

**Discussion by KingKnecht** [tailwindlabs/tailwindcss#6032] [Help] [answered]
[https://play.tailwindcss.com/eDsHr3K5AZ](https://play.tailwindcss.com/eDsHr3K5AZ)

And if opacity is not possible, can I change the color of the text when disabled?
Thx

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your question!
Much appreciated! 🙏

There are 2 issues going on:
1. The `disabled` variant is not enabled by default in AOT mode. Switching to JIT mode does enable it automatically.
2. A `disabled` class on its own doesn't exist and doesn't make the `button` have the disabled state. For that you need a `disabled` attribute.

Both fixes applied: https://play.tailwindcss.com/0ZjQN0sL6w

**Reply by KingKnecht** [reply]
@RobinMalfait nice, thx. Now I have to figure out how to enable JIT in my project. Because if I do so, it looks like all CSS is gone. But that's another story.
Thx!
